gradle.properties

```diff
@@ -1,9 +1,9 @@
-bbLearnVersion=3500.0.0
+bbLearnVersion=3700.0.0
 tomcatVersion=8.0.42
 gradleTomcatPluginVersion=2.5
 jstlVersion=1.2
 jspApiVersion=2.3.3
 servletApiVersion=4.0.1
 junitVersion=4.12
 # All building blocks / web services are re-versioned when this changes.
-artifactVersion=2.0.3
+artifactVersion=2.0.4-SNAPSHOT-D
```

Comment:
Is this intended to be a snapshot?

Reply:
for now, yes - I'll set it to 2.0.4 with a single commit to `master` and build a release for the Releases tab once this PR is buttoned up and I've ran through some smoke testing.